internal/statestore/ticket.go

```diff
@@ -255,6 +255,10 @@ func (rb *redisBackend) UpdateAssignments(ctx context.Context, req *pb.AssignTic
 		}
 	}
 
+	if len(idsI) == 0 {
+		return nil, nil, status.Error(codes.InvalidArgument, "AssignmentGroupTicketIds is empty")
```

Comment:
Isn't this test only going to match if _all_ `AssignmentGroup`s had empty `TicketIds`? It would be clearer if the message referred to that, as the closest I can see to `AssignmentGroupTicketIds` would be `AssignmentGroup.TicketIds` which would be an error appropriate for _any_  `AssignmentGroup`s having empty `TicketIds`.

Reply:
since variable `idsI` contains all `TicketIds` so this check will prevent from throwing error when there is not a single `TicketIds` is present and if any `AssignmentGroup`s having empty `TicketIds` will be silently ignored.